packages/react/src/Image/Image.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react'
+import clsx from 'clsx'
+import styles from './Image.module.css'
+
+type AspectRatio = '1:1' | '16:9' | '16:10' | '4:3' | 'custom'
+
+export interface ImageProps extends React.ImgHTMLAttributes<HTMLImageElement> {
+  ref?: React.Ref<HTMLImageElement>
```

Comment:
Could you add `src` to this list please? I think that's supposed to be required, but isn't here. 

Example from Storybook:
![Screenshot 2023-04-14 at 11 22 28](https://user-images.githubusercontent.com/13340707/232019335-651137fe-c3a2-43dc-a9f1-c237577334ad.png)